test/error-placement.js

```diff
@@ -291,6 +291,40 @@ test( "test existing non-label used as error element", function(assert) {
 	assert.noErrorFor( field );
 });
 
+test( "test aria-describedby with input names contains CSS-selector meta-characters", function() {
+	var form = $( "#testForm21" ),
+		field = $( "#testForm21\\!\\#\\$\\%\\&\\\'\\(\\)\\*\\+\\,\\.\\/\\:\\;\\<\\=\\>\\?\\@\\[\\\\\\]\\^\\`\\{\\|\\}\\~" );
```

Comment:
make sense only if you wrap the whole sting in single quotes (That's why I escaped it like that, but forgot to remove it when wrapping inside double quotes).